### ISSUE TITLE
fusion feature: mark received coins for priority fusing

### ISSUE THREAD

**Issue by emergent-reasons** [Electron-Cash/Electron-Cash#1815]
I just ran into this situation. My wallet has a bunch of utxos. When I receive a new one, there is a low chance that it gets included in any given fusion. It would be nice to have an inverse-freeze where you can mark a coin (maybe in the receive dialog and/or manually in coin list) for fusion priority. I know that could break ability to join fusions if the coin is not compatible.

**Comment by cculianu** [comment]
Yeah probably all sorts of coin tracking and coin privacy scores need to be added.  I guess this sort of falls under than umbrella.
